### PR TITLE
remove asset digest

### PR DESCRIPTION
**Pull request by tomasc**
Make sure to remove all digests from asset paths so that the assets can be found by the asset pipeline.

### CODE REVIEW

File: lib/roadie/rails/asset_pipeline_provider.rb [hunk 1]
```diff
@@ -18,7 +18,13 @@ def find_stylesheet(name)
 
       private
       def normalize_asset_name(href)
-        remove_asset_prefix href.split('?').first
+        res = remove_asset_prefix href.split('?').first
```
Comment (Mange) [inline lib/roadie/rails/asset_pipeline_provider.rb]: No test for this part.

File: lib/roadie/rails/asset_pipeline_provider.rb [hunk 2]
```diff
@@ -18,7 +18,13 @@ def find_stylesheet(name)
...
+        res = remove_asset_digest res
+        res
```
Comment (Mange) [inline lib/roadie/rails/asset_pipeline_provider.rb]: You don't need to reassign the variable here to just return it.

```ruby
  remove_asset_digest res
end
```

would suffice.

File: lib/roadie/rails/asset_pipeline_provider.rb (hunk 1 above)
Comment (tomasc) [inline lib/roadie/rails/asset_pipeline_provider.rb]: Isnt it already tested in "ignores query string and asset prefix"?

**Comment by Mange** [comment]
Since this is a pretty "dangerous" proposition (see my comment on #15), I'm not sure if I want this in or not. I see the following options:

1. Let it in as it is
  * CON: May lead to emails being very heavy to process if assets aren't properly compiled on the server and the developer would be none the wiser.
  * PRO: Less complexity for the user; helps handle complex cases; makes emails *work* when assets aren't deployed correctly instead of just crashing
2. Let it in, but with a configuration parameter
  * CON: More configuration parameters to document, test and understand.
  * PRO: Opt-in to handling complex behavior
3. Leave it out
  * CON: You cannot use `AssetPipelineProvider` in an environment with precompiled assets.
  * PRO: Roadie is never responsible for compiling assets when it shouldn't do it (`digest` is true).

I'm leaning towards alternative 1, but I'll have to sleep on it.

File: lib/roadie/rails/asset_pipeline_provider.rb (hunk 1 above)
Comment (Mange) [inline lib/roadie/rails/asset_pipeline_provider.rb]: Oh, sorry. Looking at it now makes it obvious that you didn't add anything on that line. I must've been very tired or something. :flushed:

**Comment by Mange** [comment]
I'll accept this, for version 1.1, as it requires some documentation and extra thoughts. Would you mind having your fork active for now so I can merge at a later date, or would you prefer me to create a new branch to have this merged into?

**Comment by tomasc** [comment]
I can keep this active as long as needed.

**Comment by Mange** [comment]
I fast-tracked and released this as 1.0.5 because this was needed for new Rails 4.2 apps with their new default configuration (see #36).

Thank you for your contribution! :smile: 

(Your commits were squashed; author information was kept intact)

**Pull request closed without merging**